src/foraging_gui/Foraging.py

```diff
@@ -2046,6 +2046,14 @@ def _thread_complete_timer(self):
     def _Timer(self,Time):
         '''sleep some time'''
         time.sleep(Time)
+    
+    def _EnableDisableMetadata(self,state):
```

Comment:
The name `_EnableDisableMetadata(self, state)` sounds a bit confusing. How about `_set_metadata_enabled(self, enable: bool)`, logically similar to the native setEnabled().

Reply:
sounds good! I will change it.